### ISSUE TITLE
unquoted off should not be flagged in .eslintrc.yml

### ISSUE THREAD

**Issue by erikerikson** [redhat-developer/vscode-yaml#396]
## Describe the bug

This plugin will highlight an unquoted `off` in .eslintrc.yml as violating [the schema](https://github.com/SchemaStore/schemastore/blob/master/src/schemas/json/eslintrc.json#L15) (it does not):

Example yaml:
```
root: true
rules:
  max-len: off
```

![image](https://user-images.githubusercontent.com/571200/99841882-bc6c5c80-2b23-11eb-9c5a-846fecba6989.png)

It accepts warn and error (as it should):
![image](https://user-images.githubusercontent.com/571200/99842007-ecb3fb00-2b23-11eb-97ed-17511154def9.png)
![image](https://user-images.githubusercontent.com/571200/99842039-f9385380-2b23-11eb-9680-261cd7c53ca1.png)

I suspect it might be related to yaml.tmLanguage.json [#L460](https://github.com/redhat-developer/vscode-yaml/blob/master/syntaxes/yaml.tmLanguage.json#L460) and [#L502](https://github.com/redhat-developer/vscode-yaml/blob/master/syntaxes/yaml.tmLanguage.json#L502)

## Expected Behavior

No error highlighting of the valid config

## Current Behavior

Error highlighting of valid config

## Steps to Reproduce

1. create a yaml file with the above content
2. load it into vscode with this plugin enabled
3. notice that off is highlighted with red squigglies

## Environment
- [x] Windows
- [ ] Mac
- [ ] Linux
- [ ] other (please specify)

**Comment by JPinkney** [comment]
More than likely this is because yaml parser is parsing the unquoted off as a boolean type: https://yaml.org/type/bool.html. Apparently, yaml 1.2 dropped that so it's only true/false as boolean but that would be my guess

**Comment by erikerikson** [comment]
Thank you for noticing that.  I am confused that the type doesn't seem to have been linked from the specification.  This [appears to have been discussed on the mailing list](https://sourceforge.net/p/yaml/mailman/message/12308354/).  Apologies for my level of ignorance here, I'm stepping in from "I saw valid YAML" and a teammate saying "I saw red squigglies".

3 of 4 of the top 4 search results considered off a string.  It seems there isn't a consensus but [reaching out to the yaml group at freenode.net#yaml confirmed](https://colabti.org/irclogger/irclogger_log/yaml?date=2020-11-20) your finding that as of 1.2 only true and false are to be converted to booleans.

I suppose whether this is a valid issue now rests on the version of the yaml spec this tool is intended to support (perhaps by default) which seems not my decision.  :D
